src/main/java/sirius/biz/mongo/PrefixSearchableEntity.java

```diff
@@ -110,4 +118,20 @@ private void appendSingleToken(String subToken) {
     public StringList getSearchPrefixes() {
         return searchPrefixes;
     }
+
+    /**
+     * Returns whether the search prefixes will be definitely recalculated on the next {@link BeforeSave} event
+     *
+     * @return whether the search prefixes will be definitely recalculated on the next {@link BeforeSave} event
+     */
+    public boolean isForceUpdateSearchPrefixes() {
```

Comment:
why?

Reply:
in TraceData gibts auch nen getter:
https://github.com/scireum/sirius-biz/blob/91cb8324fee3db683c3cdb5d2e360fea4d28b064/src/main/java/sirius/biz/protocol/TraceData.java#L101

aber stimmt, is unnötig